docs/reference/installation.rst

```diff
@@ -1,20 +1,30 @@
+.. index::
+    single: Installation
+    single: Configuration
+
 Installation
 ============
 
 Prerequisites
 -------------
 
-PHP 7.1 and Symfony 3.4 are needed to make this bundle work ; there are also some
-Sonata dependencies that need to be installed and configured beforehand:
+PHP ^7.2 and Symfony ^4.4 are needed to make this bundle work, there are
+also some Sonata dependencies that need to be installed and configured beforehand.
+
+Required dependencies:
+
+* `SonataCacheBundle <https://sonata-project.org/bundles/cache>`_
+* `SonataBlockBundle <https://sonata-project.org/bundles/block>`_
+* `SonataNotificationBundle <https://sonata-project.org/bundles/notification>`_
+* `SonataAdminBundle <https://sonata-project.org/bundles/admin>`_
+* `SonataDoctrineORMAdminBundle <https://sonata-project.org/bundles/doctrine-orm-admin>`_
```

Comment:
Is this dependency a requirement? What about users using ODM?

Reply:
It is not implemented on DashboardBundle

Reply:
In that case, I think we should make this situation more explicit. Even, I'd say we should have this documented as a pending task. That way, we will avoid wasting users' time and effort for who use other model managers than ORM.

Reply:
Maybe having an issue is enough? or should be document this on code in some way?

Reply:
Trying to be in the shoes of a newcomer, I'd love to know if a package will fit my needs before trying to install it. IMO, is even worse to realize that the package is not compatible with what I was thinking after the install is finished.
Given that, I guess we should have an issue to allow contributors to have clear visibility about that missing integration and, ideally, a section in the documentation exposing that.

Reply:
Issue created: https://github.com/sonata-project/SonataDashboardBundle/issues/227

Take a look at the code change, I added a note about it: `And the persistence bundle (in this case not all Sonata persistence bundles are compatible):`

wdyt? I can apply it to all bundles not supporting the doctrine mongodb persistence bundle.

Reply:
I left a small suggestion: https://github.com/sonata-project/SonataDashboardBundle/pull/225#discussion_r460426369.
I agree, we should propagate this message for all the packages which are in this same situation.